content/concepts/issued-currencies/freezes.md

```diff
@@ -60,16 +60,16 @@ An address can always enable the Global Freeze setting. However, if the address
 
 ## No Freeze
 
-The **No Freeze** feature is a setting on an address that permanently gives up the ability to freeze counterparties' issued currencies. A business can use this feature to treat its issued funds as "more like physical money" in the sense that the business cannot interfere with customers trading it among themselves.
+The **No Freeze** feature is a setting on an address that permanently gives up the ability to freeze issued currencies arbitrarily. An issuer can use this feature to treat its issued funds as "more like physical money" in the sense that the issuer cannot interfere with counterparties trading the currency among themselves.
```

Comment:
Why "arbitrarily"?

Reply:
The word "arbitrarily" represents the intended use of No Freeze and the (somewhat unintuitive) way that intended use is reflected in its functionality. Namely, that the issuer can't use individual freeze, or some kind of on-and-off use of global freeze, to temporarily stop the issued currency from being used for specific purposes or at specific times; but they can nevertheless enable a global freeze to (permanently!) stop the issued currency from being used by everyone for everything—a drastic step that's still available to help if there's a serious security issue, but is too broad/blunt/cumbersome to use for anything less.